homeassistant/components/modbus/manifest.json

```diff
@@ -2,6 +2,7 @@
   "domain": "modbus",
   "name": "Modbus",
   "documentation": "https://www.home-assistant.io/integrations/modbus",
-  "requirements": ["pymodbus==2.3.0"],
+  "requirements": ["pymodbus==2.3.0",
+		   "pyserial-asyncio==0.4"],
```

Comment:
`pymodbus` itself doesn't depend on `pyserial-asyncio`?

Reply:
Very good question. When you use serial connections it checks that it is installed, but it does not have in the requierements.txt.

I was told it is pr design, because they do not want e.g. tcp users to be demanded to include it, so instead users of pymodbus have to include it. I guess it was no seen earlier because the no async serial is included elsewhere.

Reply:
It's possible to define extras in `setup.py` so that users of the library that want to use the serial interface gets the correct requirements installed. That's the recommended approach for this problem, I think.

https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

It would be good if the library can do this, but it's ok to solve here for now.

Reply:
Please adjust the formatting in the json file. Running prettier on it should solve it.

Reply:
good idea, I will pass it on.

Reply:
Will do

Reply:
done